### PR TITLE
Add website accessibility content

### PR DESCRIPTION
**Pull request by larswmh**
image
image

---
Internal Tracking ID: [EXPOSUREAPP-11981](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-11981)

### CODE REVIEW

File: src/data/accessibility_de.json [hunk 1]
```diff
@@ -16,6 +22,16 @@
...
+            "Diese Webseite ist wegen der folgenden Ausnahmen teilweise nicht mit dem BGG und der BITV 2.0 vereinbar.",
+            "<b>Nicht barrierefreie Inhalte auf der Webseite www.coronawarn.app</b>",
+            "<p>Die nachstehend aufgeführten Inhalte sind wegen Unvereinbarkeit mit dem BGG und der BITV 2.0 nicht barrierefrei. Wir arbeiten an der baldmöglichen Beseitigung:</p>",
+            "<ul> <li>Die interaktiven Diagramme sind noch nicht in vollem Umfang für Menschen mit Sehbeeinträchtigungen zugänglich.</li> <li>Farbfehlsichtigkeit kann sich negativ auf die Lesbarkeit der Diagramme auswirken.</li> <li>Ältere PDF-Dateien der Nutzungsbedingungen und Datenschutzerklärung sind ggf. nicht barrierefrei. Später eingestellte PDF-Dateien werden in barrierefreier Version erstellt beziehungsweise schnellstmöglich durch eine solche ersetzt.</li> <li>PDF-Dateien. Diese enthalten unvollständige Alternativtexte. Die Tastaturbedienbarkeit ist nicht vollständig gegeben. Zum Teil sind Kontrastfehler enthalten.</li> <li>Benutzerdefinierte Einstellungen: Farben (z. B. Dark Mode), Kontraste und Fokus Cursor werden nicht korrekt unterstützt.</li></ul>"
```
Comment (MikeMcC399) [inline src/data/accessibility_de.json:32]:
```suggestion
            "<ul> <li>Die interaktiven Diagramme sind noch nicht in vollem Umfang für Menschen mit Sehbeeinträchtigungen zugänglich.</li> <li>Farbfehlsichtigkeit kann sich negativ auf die Lesbarkeit der Diagramme auswirken.</li> <li>Ältere PDF-Dateien der Nutzungsbedingungen und Datenschutzerklärung sind ggf. nicht barrierefrei. Später eingestellte PDF-Dateien werden in barrierefreier Version erstellt beziehungsweise schnellstmöglich durch eine solche ersetzt.</li> <li>PDF-Dateien. Diese enthalten unvollständige Alternativtexte. Die Tastaturbedienbarkeit ist nicht vollständig gegeben. Zum Teil sind Kontrastfehler enthalten.</li> <li>Benutzerdefinierte Einstellungen: Farben (z.&nbsp;B. Dark Mode), Kontraste und Fokus Cursor werden nicht korrekt unterstützt.</li></ul>"
```
to ensure that `z. B.` is not broken over two lines

Comment (larswmh) [reply src/data/accessibility_de.json]: Thanks for your suggestion! It has been committed

**Review by MikeMcC399 — APPROVED** [review]
Thanks for making the change!

**Comment by svengabr** [comment]
Changes approved.

This is a duplicate of my branch https://github.com/corona-warn-app/cwa-website/tree/accessibility-page-website-tab
Please make sure both branches are deleted when this goes live.

**Review by svengabr — APPROVED** [review]

**Review by dsarkar — APPROVED** [review]

**Comment by dsarkar** [comment]
@MikeMcC399 Thanks for review!

**Pull request merged**